hibernate-reactive-core/src/test/java/org/hibernate/reactive/containers/CockroachDBDatabase.java

```diff
@@ -25,7 +26,8 @@
 	 * TIP: To reuse the same containers across multiple runs, set `testcontainers.reuse.enable=true` in a file located
 	 * at `$HOME/.testcontainers.properties` (create the file if it does not exist).
 	 */
-	public static final CockroachContainer cockroachDb = new CockroachContainer( IMAGE_NAME )
+	public static final CockroachContainer cockroachDb = new CockroachContainer(
+			DockerImageName.parse( IMAGE_NAME ).asCompatibleSubstituteFor( "cockroachdb/cockroach:v21.2.4" ) )
```

Comment:
Could you use constant? So that we don't have to change the image and version in multiple different places?
Something like:
```
String DOCKER_REPOSITORY = "docker.io/";
public static final CockroachContainer cockroachDb = new CockroachContainer(DockerImageName.parse( DOCKER_REPOSITORY + IMAGE_NAME ).asCompatibleSubstituteFor( IMAGE_NAME  ) )
```

Maybe we can move `DOCKER_REPOSITORY` in `TestableDatabase`.

Reply:
Also, here you are keeping the version in the parameter for `asCompatibleSubstituteFor`. You are not replicating it for the other dbs. What's the difference?

Reply:
Added docker.io constant and re-worked the image name check for each db except for SQL Server which doesn't work with docker.io prefix.